src/smart-components/group/service-account/group-service-accounts.js

```diff
@@ -0,0 +1,195 @@
+import React, { useState, useEffect, useContext, useRef, Suspense, Fragment } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Outlet, useParams } from 'react-router-dom';
+import { Alert, Button } from '@patternfly/react-core';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+import Section from '@redhat-cloud-services/frontend-components/Section';
+import DateFormat from '@redhat-cloud-services/frontend-components/DateFormat';
+import { defaultSettings } from '../../../helpers/shared/pagination';
+import { TableToolbarView } from '../../../presentational-components/shared/table-toolbar-view';
+import { fetchGroup, fetchGroups } from '../../../redux/actions/group-actions';
+import { fetchServiceAccountsForGroup } from '../../../redux/actions/group-actions';
+import { getBackRoute, getDateFormat } from '../../../helpers/shared/helpers';
+import PermissionsContext from '../../../utilities/permissions-context';
+import AppLink from '../../../presentational-components/shared/AppLink';
+import useAppNavigate from '../../../hooks/useAppNavigate';
+import messages from '../../../Messages';
+import pathnames from '../../../utilities/pathnames';
+import './group-service-accounts.scss';
+
+const createRows = (data, checkedRows = []) =>
```

Comment:
Can we add a default value to the to data to avoid the `|| []` by the end of the reduce? I think its easier to ready.